src/test/kotlin/no/nb/mlt/wls/product/ProductConvertiontests.kt

```diff
@@ -0,0 +1,106 @@
+package no.nb.mlt.wls.product
+
+import no.nb.mlt.wls.core.data.Environment
+import no.nb.mlt.wls.core.data.HostName
+import no.nb.mlt.wls.core.data.Owner
+import no.nb.mlt.wls.core.data.Packaging
+import no.nb.mlt.wls.core.data.synq.SynqOwner
+import no.nb.mlt.wls.product.model.Product
+import no.nb.mlt.wls.product.payloads.ApiProductPayload
+import no.nb.mlt.wls.product.payloads.SynqProductPayload
+import no.nb.mlt.wls.product.payloads.toApiPayload
+import no.nb.mlt.wls.product.payloads.toProduct
+import no.nb.mlt.wls.product.payloads.toSynqPayload
+import org.junit.jupiter.api.Test
+
+class ProductConvertiontests {
+    val testProductPayload =
+        ApiProductPayload(
+            hostId = "mlt-test-1234",
+            hostName = HostName.AXIELL,
+            description = "Tyven skal du hete",
+            productCategory = "NONE",
+            preferredEnvironment = Environment.NONE,
+            packaging = Packaging.NONE,
+            owner = Owner.NB,
+            location = "",
+            quantity = 1.0
+        )
+
+    val testProduct =
+        Product(
+            hostId = "mlt-test-1234",
+            hostName = HostName.AXIELL,
+            description = "Tyven skal du hete",
+            productCategory = "NONE",
+            preferredEnvironment = Environment.NONE,
+            packaging = Packaging.NONE,
+            owner = Owner.NB,
+            location = "",
+            quantity = 1.0
+        )
+
+    val testSynqPayload =
+        SynqProductPayload(
+            productId = "mlt-test-1234",
+            owner = SynqOwner.NB,
+            barcode = SynqProductPayload.Barcode("mlt-test-1234"),
+            description = "Tyven skal du hete",
+            productCategory = "NONE",
+            productUom = SynqProductPayload.ProductUom(SynqProductPayload.SynqPackaging.OBJ),
+            false,
+            hostName = HostName.AXIELL.toString()
+        )
+
+    @Test
+    fun `product converts to payload`() {
+        val payload = testProduct.toApiPayload()
+        assert(payload.hostId == testProductPayload.hostId)
```

Comment:
Consider using AssertJ:  
`assertThat(payload.hostId).isEqualTo(testProductPayload.hostId) \\ I like using AssertJ for my assertions`  
or with Hamcrest:  
`assertThat(payload.hostId, equals(testProductPayload.hostId))`  
or just regular JUnit asserts:  
`assertEquals(testProductPayload.hostId, payload.hostId)`